src/py21cmfast/wrapper.py

```diff
@@ -2610,7 +2820,7 @@ def calibrate_photon_cons(
         neutral_fraction_photon_cons = np.array(neutral_fraction_photon_cons[::-1])
 
         # Construct the spline for the calibration curve
-        logger.debug("Calibrating photon conservation correction")
+        logger.info("Calibrating photon conservation correction")
         _calibrate_photon_conservation_correction(
             redshifts_estimate=z_for_photon_cons,
             nf_estimate=neutral_fraction_photon_cons,
```

Comment:
It seems that `run_coeval` and `run_lightcone` don't use the new haloes code at all? Is this intentional?

Reply:
Yeah, I haven't finalised the halo behaviour yet. Working on individual pieces before putting them into these functions.

Reply:
Cool, that's what I thought might be the case.